src/NzbDrone.Core/Indexers/Definitions/XSpeeds.cs

```diff
@@ -0,0 +1,332 @@
+        caps.Categories.AddCategoryMapping(32, NewznabStandardCategory.ConsoleNDS, "NDS Games");
+        caps.Categories.AddCategoryMapping(9, NewznabStandardCategory.Other, "Other");
+        caps.Categories.AddCategoryMapping(6, NewznabStandardCategory.PCGames, "PC Games");
+        caps.Categories.AddCategoryMapping(45, NewznabStandardCategory.Other, "Pictures");
+        caps.Categories.AddCategoryMapping(31, NewznabStandardCategory.ConsolePS4, "Playstation");
+        caps.Categories.AddCategoryMapping(71, NewznabStandardCategory.TV, "PPV");
+        caps.Categories.AddCategoryMapping(54, NewznabStandardCategory.TV, "Soaps");
+        caps.Categories.AddCategoryMapping(20, NewznabStandardCategory.TVSport, "Sports");
+        caps.Categories.AddCategoryMapping(102, NewznabStandardCategory.TVSport, "Sports FIFA World Cup");
+        caps.Categories.AddCategoryMapping(86, NewznabStandardCategory.TVSport, "Sports MotorSports");
+        caps.Categories.AddCategoryMapping(89, NewznabStandardCategory.TVSport, "Sports Olympics");
+        caps.Categories.AddCategoryMapping(88, NewznabStandardCategory.TVSport, "Sports UK Football");
+        caps.Categories.AddCategoryMapping(83, NewznabStandardCategory.Movies, "TOTM");
+        caps.Categories.AddCategoryMapping(21, NewznabStandardCategory.TVSD, "TV Boxsets");
+        caps.Categories.AddCategoryMapping(76, NewznabStandardCategory.TVHD, "TV HD Boxsets");
+        caps.Categories.AddCategoryMapping(97, NewznabStandardCategory.TVHD, "TV HECV Boxsets");
+        caps.Categories.AddCategoryMapping(47, NewznabStandardCategory.TVHD, "TV HD");
+        caps.Categories.AddCategoryMapping(96, NewznabStandardCategory.TVHD, "TV HD HEVC");
+        caps.Categories.AddCategoryMapping(16, NewznabStandardCategory.TVSD, "TV SD");
+        caps.Categories.AddCategoryMapping(7, NewznabStandardCategory.ConsoleWii, "Wii Games");
+        caps.Categories.AddCategoryMapping(43, NewznabStandardCategory.TVSport, "Wrestling");
+        caps.Categories.AddCategoryMapping(8, NewznabStandardCategory.ConsoleXBox, "Xbox Games");
+
+        return caps;
+    }
+}
+
+public class XSpeedsRequestGenerator : IIndexerRequestGenerator
+{
+    private readonly UserPassTorrentBaseSettings _settings;
+
+    public XSpeedsRequestGenerator(UserPassTorrentBaseSettings settings)
+    {
+        _settings = settings;
+    }
+
+    public IndexerPageableRequestChain GetSearchRequests(MovieSearchCriteria searchCriteria)
+    {
+        var pageableRequests = new IndexerPageableRequestChain();
+
+        pageableRequests.Add(GetPagedRequests($"{searchCriteria.SanitizedSearchTerm}", searchCriteria.FullImdbId));
+
+        return pageableRequests;
+    }
+
+    public IndexerPageableRequestChain GetSearchRequests(MusicSearchCriteria searchCriteria)
+    {
+        var pageableRequests = new IndexerPageableRequestChain();
+
+        pageableRequests.Add(GetPagedRequests($"{searchCriteria.SanitizedSearchTerm}"));
```

Comment:
Should we even be doing a search if the indexer doesn't support music? Or should we just skip it and return nothing? This would avoid a hit for nothing.